spec/GraphQL.md

```diff
@@ -122,6 +122,11 @@ explicitly in prose (e.g. "Note: ") or are set apart in a note block, like this:
 
 Note: This is an example of a non-normative note.
 
+**Lists Are Ordered**
+
+Unless otherwise stated (for example, "an unordered list"), any mention of the
+term "list" in this document indicates an ordered collection.
```

Comment:
Almost feels like we need to differentiate between syntax order and semantic (~execution?) order. In the syntax, I think I always want determined order. At execution, sometimes the order is unimportant like for input object literals. 
Don't want to bikeshed this more than needed though. This PR is a net improvement to clarity so I'm all for it 👍 .

Reply:
The coercion of input objects is explicitly unordered: https://spec.graphql.org/draft/#sec-Input-Objects.Input-Coercion; the order of a _literal_ is arguably important though, e.g. `{f(o:{a:1,b:2})}` and `{f(o:{b:2,a:1})}` are concretely different documents even though the coerced interpretation of the `o` argument is equivalent.